pkg/commands/artifact/run.go

```diff
@@ -60,12 +60,12 @@ type runner struct {
 }
 
 // NewRunner initializes Runner that provides scanning functionalities.
-func NewRunner(ctx context.Context, scanSettings settings.Config) Runner {
+func NewRunner(ctx context.Context, scanSettings *settings.Config) Runner {
```

Comment:
Not sure if this (editing the config object directly) is best practice or no; open to alternative solutions 🙏

Reply:
WDYT about putting the flag on the `Runner` instead?

Reply:
Thanks for the suggestion @didroe - much better!!